src/site/_includes/partials/topic.njk

```diff
@@ -1,21 +1,21 @@
-<h2 id="{{ topic.title | slug }}" class="no-link w-headline--three w-numbered-header w-numbered-header--horizontal">
-  {{ topic.title }}
-  <a class="w-headline-link" href="#{{ topic.title | slug }}" aria-hidden="true">#</a>
-</h2>
-<ul class="w-path-list">
-  {% for slug in topic.pathItems %}
-    {% set item = slug | findBySlug %}
-    {% if item.data.draft %}
-      {# noop #}
-    {% else %}
-      <li class="w-path-listitem">
-        <a
-          class="w-path-link"
-          href="{{ item.url | stripLanguage }}"
-        >
-          {{ item.data.title | md | safe }}
-        </a>
-      </li>
+<div class="w-topic">
```

Comment:
What do you think about doing something like this:

```
<div class="w-topic">
  <h2 id="{{ topic.title | slug }}" class="no-link w-headline--three w-topic-header">
    {{ topic.title }}
    <a class="w-headline-link" href="#{{ topic.title | slug }}" aria-hidden="true">#</a>
  </h2>
  <ul class="w-path-list">
    {% if topic.pathItems %}
      {% set section = topic %}
      {% include 'partials/item.njk' %}
    {% endif %}
  </ul>
  {% if topic.subtopics %}
    {% for section in topic.subtopics %}
      <h3 id="{{ section.title | slug }}" class="no-link w-headline--four w-subtopic-header">
        {{ section.title }}
        <a class="w-headline-link" href="#{{ section.title | slug }}" aria-hidden="true">#</a>
      </h3>
      <ul class="w-path-list">
        {% include 'partials/item.njk' %}
      </ul>
    {% endfor %}
  {% endif %}
</div>
```

I think this'll avoid the need for negative margin on the subtopic heading.